esx_service/utils/error_code.py

```diff
@@ -87,6 +88,7 @@ class ErrorCode:
     ErrorCode.TENANT_SET_ACCESS_PRIVILEGES_FAILED : "Vmgroup {0} set access privileges on datastore {1} failed with err: {2}",
     ErrorCode.TENANT_GET_FAILED : "Get vmgroup {0} failed",
     ErrorCode.TENANT_NAME_INVALID : "Vmgroup name {0} is invalid, only {1} is allowed",
+    ErrorCode.TENANT_NOT_EMPTY : "Cannot delete non empty Vmgroup {0} in Multinode mode. Remove VMs from the Vmgroup before deleting it.",
```

Comment:
Which also becomes stop containers in VMs before moving from vmgroup.

To which vmgroup should the VMs be moved to?

Pls. update documentation to specify the steps to do this.

Beginning to think these x-vmgroup (cross vmgroup) movements of VMs are actually parallels to vmotion!! If a VM has to be moved from one VM group to another we chould possibly do this (definitely not this PR):

1. Add access to the currently used datastores to the destination vmgroup (option to --retain current location of disks) - its like vmotion where the VM and its containers can continue using the same datastores as before. This is doable now.

2. Or, the VM is moved to the new VM group and the disks are migrated by initiating a storage vmotion of the in use disks (independent persistent disks cannot be migrated for now). May be with FCD this will be doable later).